cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/bridge_to_bulletin_config.rs

```diff
@@ -116,10 +117,10 @@ impl pallet_bridge_messages::Config<WithRococoBulletinMessagesInstance> for Runt
 	type BridgedHeaderChain = BridgeRococoBulletinGrandpa;
 
 	type OutboundPayload = XcmAsPlainPayload;
-
 	type InboundPayload = XcmAsPlainPayload;
-	type DeliveryPayments = ();
+	type LaneId = LegacyLaneId;
```

Comment:
you can do this as follow-up PR too

Reply:
actually it uncover one more issue, fixed here: https://github.com/paritytech/polkadot-sdk/pull/5649/commits/98aedb2dffdb7bc103df2f4aa4934d6dd8e799df

I changed it to the HashedLaneId, I don't want to remove whole Bulletin now, because we have similar setup elsewhere, so maybe it will help us there (as example, solve conflicts), but at the end we will remove it from Rococo and move to the Westend - as follow up